deploy/helm/sumologic/templates/metrics/remote-write-proxy/deployment.yaml

```diff
@@ -17,14 +17,37 @@ spec:
         checksum/config: {{ include (print $.Template.BasePath "/metrics/remote-write-proxy/configmap.yaml") . | sha256sum }}
 {{- if .Values.sumologic.podAnnotations }}
 {{ toYaml .Values.sumologic.podAnnotations | indent 8 }}
+{{- end }}
+{{- if .Values.sumologic.metrics.remoteWriteProxy.podAnnotations }}
+{{ toYaml .Values.sumologic.metrics.remoteWriteProxy.podAnnotations | indent 8 }}
 {{- end }}
       labels:
         app: {{ template "sumologic.labels.app.remoteWriteProxy.pod" . }}
         {{- include "sumologic.labels.common" . | nindent 8 }}
         {{- if .Values.sumologic.podLabels }}
         {{ toYaml .Values.sumologic.podLabels | indent 8 }}
         {{- end }}
+{{- if .Values.sumologic.metrics.remoteWriteProxy.podLabels }}
+{{ toYaml .Values.sumologic.metrics.remoteWriteProxy.podLabels | indent 8 }}
+{{- end }}
     spec:
+{{- if .Values.sumologic.metrics.remoteWriteProxy.nodeSelector }}
+      nodeSelector:
+{{ toYaml .Values.sumologic.metrics.remoteWriteProxy.nodeSelector | indent 8 }}
+{{- end }}
+{{- if or .Values.sumologic.metrics.remoteWriteProxy.affinity }}
+      affinity:
+{{ toYaml .Values.sumologic.metrics.remoteWriteProxy.affinity | indent 8 }}
+{{- end }}
+{{- if .Values.sumologic.metrics.remoteWriteProxy.tolerations }}
+      tolerations:
+{{ toYaml .Values.sumologic.metrics.remoteWriteProxy.tolerations | indent 8 }}
+{{- end }}
+      securityContext:
```

Comment:
Why not add an `if` around it?

Reply:
to be the same way like in other templates